docker-compose.yml

```diff
@@ -71,33 +98,39 @@ services:
             placement:
                 constraints:
                     - 'node.platform.os == linux'
-
-    # End
+    # End services
 
     # Start monitoring
 
     prometheus:
-        image: functions/prometheus:latest  # autobuild from Dockerfile in repo.
-        command: "-config.file=/etc/prometheus/prometheus.yml -storage.local.path=/prometheus -storage.local.memory-chunks=10000 --alertmanager.url=http://alertmanager:9093"
-        ports:
-            - 9090:9090
+        image: prom/prometheus:v2.0.0
         environment:
             no_proxy: "gateway"
+        configs:
+          - source: prometheus_config
+            target: /etc/prometheus/prometheus.yml
+          - source: prometheus_rules
+            target: /etc/prometheus/alert.rules.yml
+        command:
+          - '--config.file=/etc/prometheus/prometheus.yml'
+        #   - '-storage.local.path=/prometheus'
```

Comment:
The syntax for this is `--storage.tsdb.path` for Prom2, if we still want to store data in `/prometheus` instead of `/prometheus/data`